CMakeLists.txt

```diff
@@ -805,6 +810,10 @@ target_include_directories(${OFFSETS_LIB} PRIVATE
   kernel/include
   ${ARCH_DIR}/${ARCH}/include
   )
+
+# Make sure that LTO will never be enabled when compiling offsets.c
+set_source_files_properties(${OFFSETS_C_PATH} PROPERTIES COMPILE_OPTIONS $<TARGET_PROPERTY:compiler,prohibit_lto>)
```

Comment:
This breaks old compilers:

```
xt-xcc ERROR parsing -fno-lto:  unknown flag
```

@tejlmand how can we test this flags first, any good example to follow?

Reply:
Tentative fix submitted in #69058